.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: php
+dist: bionic
 php:
+    - 8.1.0
```

Comment:
Why do you use '8.1.0' instead of '8.0'?

Reply:
https://travis-ci.community/t/php-8-1-support/12439?u=qkdreyer

Reply:
8.1 doesn’t work whereas 8.1.0 does